src/ccall.cpp

```diff
@@ -1653,99 +1664,101 @@ static jl_cgval_t emit_ccall(jl_value_t **args, size_t nargs, jl_codectx_t *ctx)
+        return mark_or_box_ccall_result(ctx,
+            emit_bitcast(ctx, ctx.ptlsStates, lrt),
+            retboxed, rt, unionall, static_rt);
     }
     else if (is_libjulia_func(jl_threadid)) {
         assert(lrt == T_int16);
         assert(!isVa && !llvmcall);
         assert(nargt == 0);
         JL_GC_POP();
-        Value *ptls_i16 = emit_bitcast(ctx->ptlsStates, T_pint16);
+        Value *ptls_i16 = emit_bitcast(ctx, ctx.ptlsStates, T_pint16);
         const int tid_offset = offsetof(jl_tls_states_t, tid);
-        Value *ptid = builder.CreateGEP(ptls_i16, ConstantInt::get(T_size, tid_offset / 2));
-        return mark_or_box_ccall_result(
-            tbaa_decorate(tbaa_const, builder.CreateLoad(ptid)),
-            retboxed, rt, unionall, static_rt, ctx);
+        Value *ptid = ctx.builder.CreateGEP(ptls_i16, ConstantInt::get(T_size, tid_offset / 2));
+        return mark_or_box_ccall_result(ctx,
+            tbaa_decorate(tbaa_const, ctx.builder.CreateLoad(ptid)),
+            retboxed, rt, unionall, static_rt);
     }
     else if (is_libjulia_func(jl_sigatomic_begin)) {
         assert(lrt == T_void);
         assert(!isVa && !llvmcall);
         assert(nargt == 0);
         JL_GC_POP();
-        builder.CreateCall(prepare_call(gcroot_flush_func));
+        ctx.builder.CreateCall(prepare_call(gcroot_flush_func));
         Value *pdefer_sig = emit_defer_signal(ctx);
-        Value *defer_sig = builder.CreateLoad(pdefer_sig);
-        defer_sig = builder.CreateAdd(defer_sig,
+        Value *defer_sig = ctx.builder.CreateLoad(pdefer_sig);
+        defer_sig = ctx.builder.CreateAdd(defer_sig,
                                       ConstantInt::get(T_sigatomic, 1));
-        builder.CreateStore(defer_sig, pdefer_sig);
-        emit_signal_fence();
+        ctx.builder.CreateStore(defer_sig, pdefer_sig);
+        emit_signal_fence(ctx);
         return ghostValue(jl_void_type);
     }
     else if (is_libjulia_func(jl_sigatomic_end)) {
         assert(lrt == T_void);
         assert(!isVa && !llvmcall);
         assert(nargt == 0);
         JL_GC_POP();
-        builder.CreateCall(prepare_call(gcroot_flush_func));
+        ctx.builder.CreateCall(prepare_call(gcroot_flush_func));
         Value *pdefer_sig = emit_defer_signal(ctx);
-        Value *defer_sig = builder.CreateLoad(pdefer_sig);
-        emit_signal_fence();
-        error_unless(builder.CreateICmpNE(defer_sig,
-                                          ConstantInt::get(T_sigatomic, 0)),
-                     "sigatomic_end called in non-sigatomic region", ctx);
-        defer_sig = builder.CreateSub(defer_sig,
-                                      ConstantInt::get(T_sigatomic, 1));
-        builder.CreateStore(defer_sig, pdefer_sig);
+        Value *defer_sig = ctx.builder.CreateLoad(pdefer_sig);
+        emit_signal_fence(ctx);
+        error_unless(ctx,
+                ctx.builder.CreateICmpNE(defer_sig, ConstantInt::get(T_sigatomic, 0)),
```

Comment:
I agree with an extra level of indentation for line continuations of block openers - signatures, if conditions, etc. But not for call sites in normal expressions within a block.